### PR TITLE
Pullrequest bebop2 tools wifi tools

### PR DESCRIPTION
**Pull request by coppolam**
Added a set of scripts (in sw/tools/parrot/bebop) to allow Bebop 2 drones to connect to a router instead of acting as access points.

### CODE REVIEW

**Comment by dewagter** [comment]
can you rename the txt to README.md

File: sw/tools/parrot/bebop/wifi_tools/scripts/pprz.conf [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Config file for PPRZ init script
+START_PPRZ=0
```
Comment (kirkscheper) [inline sw/tools/parrot/bebop/wifi_tools/scripts/pprz.conf]: Where is this file used? Why a pprz.conf and a scripts/pprz.conf?

Comment (coppolam) [reply sw/tools/parrot/bebop/wifi_tools/scripts/pprz.conf]: Going over the scripts it actually seems that both pprz.conf and scripts/pprz.conf could be superfluous, with pprz_swarmhub.conf being sufficient. I'll double check soon and make a new pull request, either deleting them or making the distinction more clear.

**Comment by Sai-Jiang** [comment]
Hello!

I wonder whether this approach applies to Bebop2 and which firmware version it actually supports.

I have tried several approaches to convert Bebop2 to client but no one succeeds.

Thx very much

**Pull request merged**